File: URI-cpp/1046.cpp
#include <iostream>
#include <iomanip>
#include <math.h>
#include <vector>
#include <bits/stdc++.h>

using namespace std;

int main() 
{
    int i,f;

    cin >> i >> f;

    if (i == f)
    {
        cout << "O JOGO DUROU 24 HORA(S)" << endl;
    }
    else if (f > i)
    {
        cout << "O JOGO DUROU " << f-i << " HORA(S)" << endl;
    }
    else
    {
        cout << "O JOGO DUROU " << (24-i)+f << " HORA(S)" << endl;
    }
    
    return 0;
}

File: URI-py/1046.py
# -*- coding: utf-8 -*-
i, f = map(int, input().split())
if i == f:
    print("O JOGO DUROU 24 HORA(S)")
elif f > i:
    h = f - i
    print("O JOGO DUROU %d HORA(S)"%h)
elif i > f:
    h = (24-i)+f
    print("O JOGO DUROU %d HORA(S)"%h)
